lambda/custom/functions_axios.js

```diff
@@ -0,0 +1,264 @@
+
+const axios = require('axios');
+const endpoints = require('./endpoints');
+
+//Server Credentials. Follow readme to set them up.
+const oauthServiceNname = process.env.OAUTH_SERVICE_NAME;
```

Comment:
We should move all the environment variables in one place.

Reply:
Server_URL credential was moved because it was only used for endpoints while OAUTH credential is used in function so I think it is better left here